website/content/guides/1.7-plugin-upgrade/index.mdx

```diff
@@ -0,0 +1,182 @@
+github.com/hashicorp/packer/common/adapter => github.com/hashicorp/packer-plugin-sdk/adapter
+github.com/hashicorp/packer/common/bootcommand => github.com/hashicorp/packer-plugin-sdk/bootcommand
+github.com/hashicorp/packer/common/chroot => github.com/hashicorp/packer-plugin-sdk/chroot
+github.com/hashicorp/packer/common => github.com/hashicorp/packer-plugin-sdk/common
+github.com/hashicorp/packer/helper/communicator => github.com/hashicorp/packer-plugin-sdk/communicator
+github.com/hashicorp/packer/helper/config => github.com/hashicorp/packer-plugin-sdk/template/config
+github.com/hashicorp/packer/template/interpolate => github.com/hashicorp/packer-plugin-sdk/template/interpolate
+github.com/hashicorp/packer/communicator/ssh => github.com/hashicorp/packer-plugin-sdk/communicator/ssh
+github.com/hashicorp/packer/helper/communicator/sshkey => github.com/hashicorp/packer-plugin-sdk/communicator/sshkey
+github.com/hashicorp/packer/provisioner => github.com/hashicorp/packer-plugin-sdk/guestexec
+github.com/hashicorp/packer/common/json => github.com/hashicorp/packer-plugin-sdk/json
+github.com/hashicorp/packer/helper/multistep => github.com/hashicorp/packer-plugin-sdk/multistep
+steps from github.com/hashicorp/packer/common => github.com/hashicorp/packer-plugin-sdk/multistep/commonsteps
+github.com/hashicorp/packer/common/net => github.com/hashicorp/packer-plugin-sdk/net
+github.com/hashicorp/packer/packer => github.com/hashicorp/packer-plugin-sdk/packer
+github.com/hashicorp/packer/builder => github.com/hashicorp/packer-plugin-sdk/packerbuilderdata
+github.com/hashicorp/packer/packer => github.com/hashicorp/packer-plugin-sdk/pathing
+github.com/hashicorp/packer/packer/plugin => github.com/hashicorp/packer-plugin-sdk/plugin
+github.com/hashicorp/packer/common/random => github.com/hashicorp/packer-plugin-sdk/random
+github.com/hashicorp/packer/common => github.com/hashicorp/packer-plugin-sdk/retry
+github.com/hashicorp/packer/packer/rpc => github.com/hashicorp/packer-plugin-sdk/rpc
+github.com/hashicorp/packer/communicator => github.com/hashicorp/packer-plugin-sdk/sdk-internals/communicator
+github.com/hashicorp/packer/common/shell => github.com/hashicorp/packer-plugin-sdk/shell
+github.com/hashicorp/packer/common/shell-local => github.com/hashicorp/packer-plugin-sdk/shell-local
+github.com/hashicorp/packer/helper/builder/localexec => github.com/hashicorp/packer-plugin-sdk/shell-local/localexec
+github.com/hashicorp/packer/common/shutdowncommand => github.com/hashicorp/packer-plugin-sdk/shutdowncommand
+github.com/hashicorp/packer/template => github.com/hashicorp/packer-plugin-sdk/template
+github.com/hashicorp/packer/packer/tmp => github.com/hashicorp/packer-plugin-sdk/tmp
+github.com/hashicorp/packer/helper/useragent => github.com/hashicorp/packer-plugin-sdk/useragent
+github.com/hashicorp/packer/common/uuid => github.com/hashicorp/packer-plugin-sdk/uuid
+**new** github.com/hashicorp/packer-plugin-sdk/version
```

Comment:
Nice!!

Reply:
If not too handy, I think this will look better inside of a table. I looked at the UI and it looks a bit confusing to look at and get information from it.